build.py

```diff
@@ -71,7 +71,7 @@
 CORE_BACKENDS = ['tensorrt', 'ensemble']
 NONCORE_BACKENDS = [
     'tensorflow1', 'tensorflow2', 'onnxruntime', 'python', 'dali', 'pytorch',
-    'openvino', 'fil'
+    'openvino', 'fil', 'tflite'
```

Comment:
Instead if using 'tflite' can we instead refer to the backend as 'armnn' or some similar arm-specific term. Similarly the backend should be called 'armnn_backend" or similar. We don't want to represent as tflite as we are not claiming general tflite support.

Reply:
Well this backend is for TFLite, and enables the use of the ArmNN and XNNPACK delegate to accelerate any operations which they each implement respectively. If the delegates are not used the default TFLite runtime is used. See: https://gitlab.com/arm-research/smarter/tflite_backend